src/Settings/VariableIterator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @file
+ * Contains SiteAudit\Settings\VariableIterator
+ */
+
+namespace SiteAudit\Settings;
+
+
+class VariableIterator extends SettingsIterator {
+
+  /**
+   * Determine if a variable is configured correctly.
+   *
+   * @return bool
+   */
+  public function valid() {
+    $actual = $this->drush()->getVariable($this->key());
+    return $this->current() == $actual;
```

Comment:
A double equals will help with the `TRUE` vs `1` etc that may be in the settings.